app/views/saml_idp/idp/saml_post.html.erb

```diff
@@ -2,13 +2,27 @@
 <html>
   <head>
     <meta charset="utf-8">
+    <%= csp_meta_tag %>
```

Comment:
for some reason this method is only callable from rails 5.2
it breaks tests in all other releases